### ISSUE TITLE
Create data source for ACL policy

### ISSUE THREAD

**Issue by lgfa29** [terraform-providers/terraform-provider-nomad#82]
# nomad_acl_policy

Get information on an [ACL policy](https://www.nomadproject.io/api/acl-policies.html#read-policy). 

## Example usage

```hcl
data "nomad_acl_policy" "my_policy" {
  name = "my-policy"
}
```

## Argument reference

* `name`: `(string)` Name of the ACL policy.

## Attributes reference

* `name`: `(string)` Name of the ACL policy.
* `description`: `(string)` ACL policy description.
* `rules`: `(string)` Policy rules in HCL or JSON format.

**Comment by ddreier** [comment]
I believe that I have this mostly done. Will submit a PR soon.

**Comment by cgbaker** [comment]
closed by #89
